acquia/acquia_cms
# ACMS-1741: Remove ACMS Site Studio Config Management module

**Motivation**
Fixes #ACMS-1741

**Proposed changes**
Remove ACMS Site Studio Config Management module.

**Alternatives considered**
NA

**Testing steps**
- Install site with this branch
- install ACMS Site Studio
- import packages
- Execute `drush cex`
- Verify config imported to `config/default/sync` and site studio packages to `config/default/sitestudio`

**Merge requirements**
